### PR TITLE
Add family to stats printout

### PR DESCRIPTION
**Pull request by thhous-msft**

### CODE REVIEW

File: src/quicreach.cpp [hunk 1]
```diff
@@ -174,19 +174,27 @@ bool TestReachability(const ReachConfig& Config) {
...
-                        (uint32_t)Connection.Stats.SendTotalPackets,
-                        (uint32_t)Connection.Stats.RecvTotalPackets,
-                        (uint32_t)Connection.Stats.SendTotalBytes,
-                        (uint32_t)Connection.Stats.RecvTotalBytes,
-                        Amplification,
-                        Connection.Stats.HandshakeClientFlight1Bytes,
-                        Connection.Stats.HandshakeServerFlight1Bytes,
-                        TooMuch ? '!' : (MultiRtt ? '*' : ' '));
+            if (Config.PrintStatistics) {
+                QuicAddr RemoteAddr;
```
Comment (nibanks) [inline src/quicreach.cpp:178]: Put this stuff in the `QUIC_CONNECTION_EVENT_CONNECTED` event callback instead.

File: src/quicreach.cpp [hunk 2]
```diff
@@ -174,19 +179,21 @@ bool TestReachability(const ReachConfig& Config) {
...
-                        HandshakeTime / 1000, HandshakeTime % 1000,
-                        (uint32_t)Connection.Stats.SendTotalPackets,
-                        (uint32_t)Connection.Stats.RecvTotalPackets,
-                        (uint32_t)Connection.Stats.SendTotalBytes,
-                        (uint32_t)Connection.Stats.RecvTotalBytes,
-                        Amplification,
-                        Connection.Stats.HandshakeClientFlight1Bytes,
-                        Connection.Stats.HandshakeServerFlight1Bytes,
-                        TooMuch ? '!' : (MultiRtt ? '*' : ' '));
+            if (Config.PrintStatistics) {
```
Comment (nibanks) [inline src/quicreach.cpp:182]: Just leave the `{` off

Comment (thhous-msft) [reply src/quicreach.cpp:182]: We should never have an if statement like that without the brace. The statement contained inside is WAY too large to make it easy to comprehend.

Comment (thhous-msft) [reply src/quicreach.cpp:182]: In fact I'd prefer to go fix all the other cases in here too. There is no reason to skip braces. Doing so just makes code harder to read.

Comment (nibanks) [reply src/quicreach.cpp:182]: I was trying mostly to follow the Linux style a bit more.

Comment (thhous-msft) [reply src/quicreach.cpp:182]: Thats a disgusting coding style... But sure whatever

**Review by nibanks — APPROVED** [review]

File: src/quicreach.cpp (hunk 2 above)
Comment (nibanks) [reply src/quicreach.cpp]: I'm not going to argue against you. 😄 You know I like our style, but I was trying to make this more attractive for Linux dev contributors.

**Pull request merged**